src/node_dotenv.h

```diff
@@ -0,0 +1,37 @@
+#ifndef SRC_NODE_DOTENV_H_
+#define SRC_NODE_DOTENV_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "util-inl.h"
+
+#include <map>
+#include <optional>
+
+namespace node {
+
+class Dotenv {
+ public:
+  Dotenv() = default;
+  Dotenv(const Dotenv& d) = default;
+  Dotenv(Dotenv&& d) noexcept = default;
+  Dotenv& operator=(Dotenv&& d) noexcept = default;
+  Dotenv& operator=(const Dotenv& d) = default;
+  ~Dotenv() = default;
+
+  void parse(const std::string_view path);
+  void assignNodeOptionsIfAvailable(std::string* node_options);
+  void set_env(Environment* env);
+  static std::optional<std::string> GetPathFromArgs(
+      const std::vector<std::string>& args);
```

Comment:
This is a surprising mix of naming conventions for public functions: `snake_case`, `camelCase`, and `PascalCase` within three lines of code. Due to V8, we generally follow [Google's style guide](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules), which uses `PascalCase` for function names. (We do sometimes deviate from that and follow C++ standard library naming conventions, but there is no apparent reason here to follow different conventions within the same class.)